ang/civicase/case/search/directives/search.directive.js

```diff
@@ -19,7 +19,10 @@
    * Controller Function for civicase-search directive
    */
   module.controller('civicaseSearchController', function ($scope, $rootScope, $timeout,
-    crmApi, getSelect2Value, ts, CaseStatus, CaseTypeCategory, CaseType, CustomSearchField) {
+    crmApi, getSelect2Value, ts, CaseStatus, CaseTypeCategory, CaseType, currentCaseCategory, CustomSearchField) {
+    var DEFAULT_CASE_FILTERS = {
```

Comment:
vars and $scope properties are mixed. lets separate them out.
1. single line vars
2. multi line vars
3. single line $scope vars
4. multi line $scope vars